cgo_flags.go

```diff
@@ -3,7 +3,7 @@
 package jemalloc
 
 // #cgo CFLAGS: -Iinternal/include
-// #cgo linux CFLAGS: -DJEMALLOC_PURGE_MADVISE_DONTNEED
-// #cgo darwin CFLAGS: -DJEMALLOC_OSATOMIC -DJEMALLOC_OSSPIN -DJEMALLOC_PURGE_MADVISE_FREE -DJEMALLOC_HAVE_ISSETUGID
-// #cgo darwin CFLAGS: -DJEMALLOC_ZONE -DJEMALLOC_ZONE_VERSION=8
+// #cgo darwin CFLAGS: -Idarwin_includes -Idarwin_includes/jemalloc/internal
+// #cgo linux CFLAGS: -Ilinux_includes -Ilinux_includes/jemalloc/internal
+// #cgo linux LDFLAGS: -lm
```

Comment:
do you not need `-lm` on darwin? It was needed on linux after I enabled profiling and the `log()` calls in `prof.c` came into play